src/main/java/be/seeseemelk/mockbukkit/entity/EntityMock.java

```diff
@@ -1229,8 +1230,7 @@ public boolean fromMobSpawner()
 	@Override
 	public CreatureSpawnEvent.@NotNull SpawnReason getEntitySpawnReason()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.spawnReason;
```

Comment:
Should these changes be part of the pull request? What I understand, they do not relate to AllayMock.

(Personally it's fine for me as is)